File: src/problems_all/709_to_lower_case/709_to_lower_case.c
/*
https://leetcode.com/problems/to-lower-case/description/

Runtime: 4 ms
Memory: 6.5 MB
*/

#include <stdio.h>

char *toLowerCase(char *str)
{
    for (char *c = str; *c; c++)
        if (*c >= 'A' && *c <= 'Z')
            *c = (*c - 'A') + 'a';

    return str;
}

int main()
{
    char *s = "Hello";
    char *result = toLowerCase(s);
    printf("%s\n", result);

    return 0;
}

File: src/problems_all/709_to_lower_case/709_to_lower_case.go
/*
https://leetcode.com/problems/to-lower-case/description/

Runtime: 0 ms
Memory: 2 MB
https://leetcode.com/problems/to-lower-case/submissions/879278971/
*/

package main

import (
	"fmt"
	"strings"
)

func main() {
	s := "Hello"
	result := toLowerCase(s)
	fmt.Println(result)
}

func toLowerCase(s string) string {
	return strings.ToLower(s)
}
